ui/app/AppLayouts/Shell/ShellDockButton.qml

```diff
@@ -0,0 +1,136 @@
+import QtQuick 2.15
+import QtQuick.Controls 2.15
+
+import StatusQ.Core 0.1
+import StatusQ.Controls 0.1
+import StatusQ.Components 0.1
+import StatusQ.Core.Theme 0.1
+import StatusQ.Popups 0.1
+
+import utils 1.0
+
+ToolButton {
```

Comment:
can we add object name for button delegate? Something like ShellButton_Market , etc?